CompanyEmployees.Infrastructure.Persistence/RepositoryContext.cs

```diff
@@ -14,8 +14,10 @@ public RepositoryContext(DbContextOptions options)
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.ApplyConfiguration(new CompanyConfiguration());
+            modelBuilder.ApplyConfiguration(new AttendanceConfiguration());
         }
 
```

Comment:
Consider removing the nullable marker (`?`) from `DbSet<Attendance>` to align with common EF Core practices. EF Core expects non-nullable `DbSet` properties since it handles initialization internally. Keeping them non-nullable helps avoid unnecessary null checks and potential confusion.

**Suggested change**
```suggestion
public DbSet<Attendance> Attendances { get; set; }
```